cli/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ _Released 03/28/2023 (PENDING)_
 **Misc:**
 
  - Made some minor styling updates to the Debug page. Addresses [#26041](https://github.com/cypress-io/cypress/issues/26041).
+ - Capture the [Buildkite](https://buildkite.com/) CI provider's environment variable `BUILDKITE_RETRY_COUNT` to handle CI retries in the Cloud. Addressed in [25750](https://github.com/cypress-io/cypress/pull/25750).
+ - Capture the [Azure](https://azure.microsoft.com/) CI provider's environment variable [`SYSTEM_PULLREQUEST_PULLREQUESTNUMBER`](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services) to display the value in the Cloud. Addressed in [26215](https://github.com/cypress-io/cypress/pull/26215)
```

Comment:
I expect this to be either a bug fix or feature enhancement, not listed as a misc update

Reply:
I just updated it to be under Bugfixes

Reply:
I just updated it to be under Bugfixes